ui/src/app/components/plans/test-plan-editor/test-plan-suite-machine-selection.component.ts

```diff
@@ -136,6 +136,12 @@ export class TestPlanSuiteMachineSelectionComponent extends BaseComponent implem
       if(res) {
         if(this.filter == 'All') {
           this.testSuiteList = this.versionFilter? [...this.testSuiteList.filter(item=> item.workspaceVersionId != this.versionFilter.id), ...res] : res;
+          if(!this.hasMixedAppVersion){
```

Comment:
can we combine the if/else block and set the chcked state by the condition statement?